---[LOGGER MSG START]---
Startup - 09:47:40 : woensdag 22 mei 2019
>Starting server...
>Server is up and running!
---[LOGGER  MSG  END]---

---[LOGGER MSG START]---
[CheckLogin] - 09:47:43 : woensdag 22 mei 2019
>Checking log in for user JMuris
>Something went wrong: System.Data.SqlClient.SqlException (0x80131904): Cannot open database "TrustMeDb" requested by the login. The login failed.
Login failed for user 'DBMASTER'.
   at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, Boolean applyTransientFaultHandling, String accessToken)
   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at WerkelijkWaar.Classes.DatabaseQueries.CheckLogin(String username, String password) in C:\Users\Justin Muris\source\repos\WerkelijkWaar\WerkelijkWaar\Classes\DatabaseQueries.cs:line 211
ClientConnectionId:7b7d77ac-37ef-4460-abdf-21d8bb9178a4
Error Number:4060,State:1,Class:11
---[LOGGER  MSG  END]---

---[LOGGER MSG START]---
Startup - 09:48:13 : woensdag 22 mei 2019
>Starting server...
>Server is up and running!
---[LOGGER  MSG  END]---

---[LOGGER MSG START]---
[CheckLogin] - 09:48:16 : woensdag 22 mei 2019
>Checking log in for user JMuris
---[LOGGER MSG START]---
[RetrieveUser] - 09:48:16 : woensdag 22 mei 2019
>Attempting to retrieve user with id 4
>Found user with id 4
---[LOGGER  MSG  END]---

---[LOGGER MSG START]---
[CheckLogin] - 09:48:28 : woensdag 22 mei 2019
>Checking log in for user Admin
---[LOGGER MSG START]---
[RetrieveUser] - 09:48:28 : woensdag 22 mei 2019
>Attempting to retrieve user with id 3
>Found user with id 3
---[LOGGER  MSG  END]---

---[LOGGER MSG START]---
[CheckLogin] - 09:48:50 : woensdag 22 mei 2019
>Checking log in for user System
---[LOGGER MSG START]---
[RetrieveUser] - 09:48:50 : woensdag 22 mei 2019
>Attempting to retrieve user with id 1
>Found user with id 1
---[LOGGER  MSG  END]---

---[LOGGER MSG START]---
[CheckLogin] - 09:49:25 : woensdag 22 mei 2019
>Checking log in for user ÜberAdmin
>Log in not found for user ÜberAdmin
---[LOGGER  MSG  END]---

---[LOGGER MSG START]---
[CheckLogin] - 09:49:36 : woensdag 22 mei 2019
>Checking log in for user UberAdmin
---[LOGGER MSG START]---
[RetrieveUser] - 09:49:36 : woensdag 22 mei 2019
>Attempting to retrieve user with id 2
>Found user with id 2
---[LOGGER  MSG  END]---

